### PR TITLE
Bug 1139654 - iOS home page promo

### PR DESCRIPTION
**Pull request by craigcook**

### CODE REVIEW

**Comment by flodolo** [comment]
Strings extracted locally.

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 1]
```diff
@@ -124,15 +124,27 @@
...
+      {% elif l10n_has_tag('hello_promo') %}
+        <li id="promo-1" class="item promo-large-portrait firefox-hello" data-name="Meet Firefox Hello" tabindex="0">
+          <h2 class="primary go">{{ _('Meet<br> Firefox Hello') }}</h2>
+          <a class="panel-link" href="{{ url('firefox.hello') }}">
             <div class="secondary">
               <h3>{{ _('The features <br>you love. <br>The privacy <br>you trust.') }}</h3>
               <p class="more">{{ _('Learn about <br>Firefox for Android') }}</p>
             </div>
           </a>
         </li>
```
Comment (jpetto) [inline bedrock/mozorg/templates/mozorg/home/home.html]: The first part of this promo is Hello, the second Android. What the?

If Hello is intended, making sure we're okay with no promo in this spot if both `l10n_has_tag` conditionals happen to be falsy.

Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Doh, looks like maybe a botched merge. Will fix. Yay code review!

**Comment by jpetto** [comment]
Looking good! r+ :train2:

(Get on the train, Travis.)

**Pull request merged**